index.html

```diff
@@ -69,142 +74,150 @@ <h2 class="yoda">
-          </p>
-          <ul class="grid-list2">
-            <li class="list2">HTML</li>
-            <li class="list2">CSS</li>
-            <li class="list2">Javascript</li>
-          </ul>
-          <h3 class="seeproject">See Project</h3>
+        <div class="-grid-container">
+          <img src="images/Snapshoot Portfolio new 3.svg" alt="" class="gridpicture3" />
+          <div class="grid-container-text">
+            <h2 class="tonic">Facebook 360</h2>
+            <ul class="grid-list">
+              <li class="canopy-list">FACEBOOK</li>
+              <li class="back-list">Full Stack Dev</li>
+              <li class="year-list">2015</li>
+            </ul>
+            <p class="gridtext">
+              Exploring the future of media in Facebook's first Virtual Reality
+              <br />
+              app; a place to discover and enjoy 360 photos and videos on <br />
+              Gear VR.
+            </p>
+            <ul class="grid-list2">
+              <li class="list2">html</li>
+              <li class="list2">Ruby on rails</li>
+              <li class="list2">CSS</li>
+              <li class="list2">Javascript</li>
+            </ul>
+            <button class="seeproject">See Project</button>
+          </div>
         </div>
 
-        <div class="grid-container">
-          <img src="Snapshoot Portfolio(3).svg" alt="" class="gridpicture" />
-          <h2 class="tonic">
-            Multi-Post <br />
-            Stories
-          </h2>
-          <ul class="grid-list">
-            <li class="canopy-list">CANOPY</li>
-            <li class="back-list">Back End Dev</li>
-            <li class="year-list">2015</li>
-          </ul>
-          <p class="gridtext">
-            A daily selection of privately <br />
-            personalized reads; no accounts or <br />
-            sign-ups required.
-          </p>
-          <ul class="grid-list2">
-            <li class="list2">HTML</li>
-            <li class="list2">CSS</li>
-            <li class="list2">Javascript</li>
-          </ul>
-          <h3 class="seeproject">See Project</h3>
+        <div class="third-grid-container">
+          <div class="grid-container-text">
+            <h2 class="tonic">Uber Navigation</h2>
+            <ul class="grid-list">
+              <li class="canopy-list">Uber</li>
+              <li class="back-list">Lead Developer</li>
+              <li class="year-list">2018</li>
+            </ul>
+            <p class="gridtext">
+              A smart assistant to make driving more safe, efficient, and fun
+              <br />
+              by unlocking your most expensive computer: your car.
+            </p>
+            <ul class="grid-list2">
+              <li class="list2">html</li>
+              <li class="list2">Ruby on rails</li>
+              <li class="list2">CSS</li>
+              <li class="list2">Javascript</li>
+            </ul>
+            <button class="seeproject">See Project</button>
```

Comment:
> - [ ] Hello @CrackedYoda, I observed on the desktop screen that you've incorporated the hover button state, but you haven't included the pressed button state as the code reviewer requested.😌 It would be appreciated if you could address this. Doing so will improve the user experience and ensure your design aligns with the Figma design template.👍

Reply:
i already implemented it